src/components/ebay-textbox/examples/7-multiline/template.marko

```diff
@@ -0,0 +1 @@
+<ebay-textbox multiline />
```

Comment:
Please name the example folders with a leading `0`, as in `01 02 03`. Also, fix the numbering so `06` isn't skipped.